cantonese/siuming.md

```diff
@@ -0,0 +1,7 @@
+小明係一個有為青年，不過就比較大頭蝦，成日唔記得事情。有一日，佢要搭飛機去美國讀書，同
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.